src/Swarm/Game/Step.hs

```diff
@@ -271,6 +277,15 @@ stepCEK cek = case cek of
     if wakeupTime == time
       then stepCEK cek'
       else return cek
+  Out v (FImmediate wf rf : c) -> do
```

Comment:
Let's stay consistent with the rest of the code and use `k` for the continuation instead of `c`.  It might seem like a bit of a weird convention if you've never seen it before, but it is extremely common to represent continuations with the variable `k` (perhaps because the word for it starts with k in German?).  And `c` is used for lots of other things already.

Reply:
Ich habe es geändert.